uniffi_bindgen/src/bindings/python/templates/ErrorTemplate.py

```diff
@@ -3,47 +3,42 @@
 # {{ type_name }}
 # We want to define each variant as a nested class that's also a subclass,
 # which is tricky in Python.  To accomplish this we're going to create each
-# class separated, then manually add the child classes to the base class's
+# class separately, then manually add the child classes to the base class's
 # __dict__.  All of this happens in dummy class to avoid polluting the module
 # namespace.
-class UniFFIExceptionTmpNamespace:
-    class {{ type_name }}(Exception):
-        pass
+class {{ type_name }}(Exception):
+    pass
+
+{{ type_name }}Temp = {{ type_name }}
```

Comment:
- I think this name should be deleted at the end of the definitions to avoid polluting the namespace
- The name should be something that's guaranteed not to clash with any other user names.  Users may define a `Foo` and `FooTemp` class.  It's not 100% foolproof, but I usually use a prefix that starts with "uniffi" (maybe `UniFFiTemp{{ type_name }}` in this case.

Reply:
Did both.